crates/proposer-service/src/config.rs

```diff
@@ -18,6 +11,25 @@ const DEFAULT_PROPOSER_SERVICE_ENDPOINT: &str = "http://proposer-mock-rpc:3000";
 /// The default url endpoint for the grpc cluster service
 const DEFAULT_SP1_CLUSTER_ENDPOINT: &str = "https://rpc.production.succinct.xyz/";
 
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
+#[serde(rename_all = "kebab-case")]
+pub struct ProposerServiceConfig {
+    pub client: ProposerClientConfig,
```

Comment:
This `ProposerClientConfig` should maybe be defined in the `proposer-client` crate